hw/ip/clkmgr/doc/theory_of_operation.md

```diff
@@ -213,6 +213,7 @@ The life cycle controller only requests the io clock input to be switched.
 When the life cycle controller requests external clock, a request signal `lc_clk_byp_req_i` is sent from `lc_ctrl` to `clkmgr`.
 `clkmgr` then forwards the request to `ast` through `io_clk_byp_req_o`, which performs the actual clock switch and is acknowledged through `io_clk_byp_ack_i`.
 When the clock switch is complete, the clock dividers are stepped down by a factor of 2 and the life cycle controller is acknowledged through `lc_clk_byp_ack_o`.
+E.g. if the internal clock has a nominal frequency of 96MHz, the external clock is expected to have a frequency of only 48MHz after the clock switch is complete.
```

Comment:
This seems unclear. Perhaps pointing to the frequencies table below will help. The nominal IO clock frequency "is" 96 HMz, and the external clock frequency "must be" 48 MHz. Stepping down the clock dividers causes the indicated frequencies for io_div2 and io_div4.

Reply:
Good idea, let me do that.